pkg/lambdaworks/lambdaworks.go

```diff
@@ -72,18 +74,26 @@ func (felt Felt) ToU64() (uint64, error) {
 	}
 }
 
+// Gets a Felt representing 1.
+func FeltOne() Felt {
+	var result C.felt_t
+	C.one(&result[0])
+	return fromC(result)
+}
+
 // Gets a Felt representing 0.
 func FeltZero() Felt {
 	var result C.felt_t
 	C.zero(&result[0])
 	return fromC(result)
 }
 
-// Gets a Felt representing 1.
-func FeltOne() Felt {
-	var result C.felt_t
-	C.one(&result[0])
-	return fromC(result)
+func (f Felt) Zero() Felt {
+	return FeltZero()
+}
+
+func (f Felt) One() Felt {
+	return FeltOne()
```

Comment:
Are this functions necessary? We already have `FeltZero` and `FeltOne`

Reply:
In a past PR they were demanded at some point, but I agree they don't seem to be very useful right now

Reply:
I changed them to `FeltZero` & `FeltOne` in one of my PR's, it looks like they weren't removed after merging main

Reply:
Solved here: https://github.com/lambdaclass/cairo-vm.go/pull/52/commits/b3e67381c647c6e3539d593e3569d4de9e53aa91